tensorflow/contrib/rnn/python/ops/core_rnn_cell_impl.py

```diff
@@ -83,12 +83,14 @@ def call(self, inputs, state):
 class GRUCell(RNNCell):
   """Gated Recurrent Unit cell (cf. http://arxiv.org/abs/1406.1078)."""
 
-  def __init__(self, num_units, input_size=None, activation=tanh, reuse=None):
+  def __init__(self, num_units, input_size=None, activation=tanh, reuse=None,
+               initializer=None):
     super(GRUCell, self).__init__(_reuse=reuse)
     if input_size is not None:
       logging.warn("%s: The input_size parameter is deprecated.", self)
     self._num_units = num_units
     self._activation = activation
+    self._initializer = initializer
```

Comment:
As much as possible, you should use the same keywords (for the same features) to avoid API fragmentation. It is preferable not to have 2 redundant APIs where the same things are configured via similar-but-not-quite-the-same keywords.

In TF layers, initializers should simply be functions (this is also the case in Keras, but we also allow function names as strings for convenience).